bbc/simorgh
# Media page metadata and propTypes tweaks

Resolves #2750

**Overall change:** 
- Adds dari and persian radio to give an example of an rtl page
- Adds some propType definitions for media page
- Adds a type field for openGraph and schemaOrg for the media page 

**Code changes:**

- Adds propTypes for the metadata, content, promo, and mediaPageData
- Adds the og and schema `type` values for media page (`website` and `RadioChannel` respectively)
- Changes `bbc_afaanoromoo_radio` payload to have `externalId` as `bbc_oromo_radio` && fixes typo in uri name
- Changes Amharic payload to not have Tigrinya 
- Loops over the Ares blocks of to be more resilient if the headline and paragraph blocks are in a different order in the payload (this will be useful when we get round to rendering more blocks) 
- Refactors `getLink()` so that it supports more than articles and frontPage assets
- Adds metadata test for the liveradio page 

---


TODO:
- add `metadata.summary` to all payloads and tell Ares this should be populated. 
- check fixture data
